plugins/setoid_ring/newring.ml

```diff
@@ -73,12 +73,12 @@ let lookup_map map =
   with Not_found ->
     CErrors.user_err ~hdr:"lookup_map" (str"map "++qs map++str"not found")
 
-let protect_red map env sigma c0 =
-  let evars ev = Evarutil.safe_evar_value sigma ev in
+let protect_red map env evd c0 =
```

Comment:
Why the renaming? I was under the impression that the current preferred naming scheme was `sigma` rather than `evd`.

Reply:
This one slipped after the squash, I'll roll it back.